.ci/jenkins-execute-script.py

```diff
@@ -73,6 +73,7 @@ def prepare_pull_request_build(host):
     build_cmd += 'bundle exec rake build && '
     build_cmd += 'bundle exec rake get_adb[\'libvirt\'] && '
     build_cmd += 'bundle exec rake features CUCUMBER_OPTS=\'-p ci\' PROVIDER=libvirt'
```

Comment:
Missing '&&' in the end. Also I think it is more natural to first run the unit tests and then the acceptance tests.